src/mocks.ts

```diff
@@ -47,11 +50,35 @@ export class NavMock {
 }
 
 export class PlatformMock {
-  public ready(): any {
-    return new Promise((resolve: Function) => {
-      resolve();
+  public ready(): Promise<{String}> {
+    return new Promise((resolve) => {
+      resolve('READY');
     });
   }
+
+  public registerBackButtonAction(fn: Function, priority?: number): Function {
+    return (() => true);
+  }
+
+  public hasFocus(ele: HTMLElement): boolean {
+    return true;
+  }
+
+  public doc(): HTMLDocument {
+    return document;
+  }
+
+  public registerListener(ele: any, eventName: string, callback: any): Function {
+    return (() => true);
+  }
+
+  public win(): Window {
+    return window;
+  }
+
+  public raf(callback: any): number {
```

Comment:
This file contains all the mocks used for testing. Honestly, I don't know what much of the methods in there do, I simply know that some stuff in ionic depend on them and I have to mock them to make the tests work. So I am not sure it is necessary to describe what the individual method do, but it is true that a general comment about what the contain of the file does would be helpful. Do you agree?